src/main/src/test/java/com/shevtsov/dao/impl/ClientDBDaoTest.java

```diff
@@ -0,0 +1,156 @@
+package com.shevtsov.dao.impl;
+
+import com.shevtsov.dao.ClientDao;
+import com.shevtsov.dao.DBConnection;
+import com.shevtsov.domain.Client;
+import org.junit.*;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class ClientDBDaoTest {
```

Comment:
By default *Test is unit test and it should be without database interaction
*IT is integration test and it could access to database

Reply:
fixed and added separate phase for IT